spec/forms/developer_search_form.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe DeveloperSearchForm do
+  describe '#search' do
+
+    let(:language) { create :language }
+    let(:programming_language) { create :programming_language }
+
+    before do
+      create(:developer, programming_languages: [programming_language], languages: [language])
+      create :developer
```

Comment:
https://github.com/bbatsov/ruby-style-guide#method-invocation-parens

Reply:
@ntrung2006 
The same thing here.
```ruby
create(:developer)
```

Reply:
Fixed!